build_psa_compliance.py

```diff
@@ -37,7 +37,7 @@
                     "tgt_ff_tfm_musca_a"],
```

Comment:
Could we remove this please, in a new commit? `ARM_MUSCA_A1` is not supported on Mbed OS 6.0

Reply:
Yes, separate PR

Reply:
Created https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/issues/28